cmake/modules/CMakeLists.txt

```diff
@@ -1,7 +1,23 @@
 
 set(DISPATCH_EXPORTS_FILE ${CMAKE_CURRENT_BINARY_DIR}/dispatchExports.cmake)
 configure_file(dispatchConfig.cmake.in
-  ${CMAKE_CURRENT_BINARY_DIR}/dispatchConfig.cmake)
+  ${CMAKE_CURRENT_BINARY_DIR}/dispatchConfig.cmake
+  @ONLY)
+
+if(LibRT_FOUND)
+  file(COPY FindLibRT.cmake DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+endif()
 
 get_property(DISPATCH_EXPORTS GLOBAL PROPERTY DISPATCH_EXPORTS)
 export(TARGETS ${DISPATCH_EXPORTS} FILE ${DISPATCH_EXPORTS_FILE})
+
+if(NOT ENABLE_SWIFT)
```

Comment:
Why the `NOT ENABLE_SWIFT` condition for this?

Reply:
I've adapted only the native library part. I don't have swift, I use dispatch as a native library in a C++ project and a cmake config for it would make consumption way easier.